scripts/install.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+set -e
+
+RELEASES_URL="https://github.com/shipyardbuild/shipyard-cli/releases"
+
+last_version() {
+    curl --silent --location --fail \
+    --output /dev/null --write-out %{url_effective} ${RELEASES_URL}/latest |
+    grep -Eo '[0-9]+\.[0-9]+\.[0-9]+$'
+}
+
+main() {
+    default_dir=/usr/local/bin
+
+    case $(uname -m) in
```

Comment:
I tried this in a Linux container on an arm64 machine, and ARCH is not being set.

Maybe slightly different logic would cover more cases?
```
ARCH=$(uname -m)
case $ARCH in
aarch64) ARCH="arm64";;
x86_64) ARCH="amd64";;
esac
```